security/json_login_setup.rst

```diff
@@ -18,7 +18,7 @@ First, enable the JSON login under your firewall:
             firewalls:
                 main:
                     anonymous: ~
-                    json_login:
+                    json-login:
```

Comment:
Did you experience any issues when using the notation using the underscore?

Reply:
It did not work with the underscore.
In JsonLoginFactory.php, the getKey function returns 'json-login', this is what made me look at that solution.

Reply:
Ok, forget about this.
Seems i changed many things at once... it now works both ways.